### PR TITLE
Add certificate update to Dockerfile

### LINKED ISSUE #3: Docker container can no longer load some https URLs
I'm having a hard time debugging why, but it looks like your docker container no longer loads https:

     docker run -p3000:3000 eliksir/screenie-server &
     [1] 51718
     Screenie server started on port 3000

    andy:~ andrey$ curl -I http://localhost:3000/?url=https://sigma-static-files.imgix.net/uspalogo.png
    HTTP/1.1 404 Not Found
    Content-Type: text/plain; charset=utf-8
    Content-Length: 9
    Date: Sat, 01 Jul 2017 03:19:19 GMT
    Connection: keep-alive

    andy:~ andrey$ curl -I http://localhost:3000/?url=http://sigma-static-files.imgix.net/uspalogo.png
    HTTP/1.1 200 OK
    Content-Type: image/jpeg
    Content-Length: 27874
    Date: Sat, 01 Jul 2017 03:19:23 GMT
    Connection: keep-alive

### PR DESCRIPTION
**Pull request by anfedorov**
Fixes #3 using instructions from this SO answer:
https://stackoverflow.com/a/33353762/8016079

### CODE REVIEW

**Comment by anfedorov** [comment]
if you'd like to keep the change even more minimal, this works, as well: 
```
FROM node:7.7.2-alpine

ENV SCREENIE_VERSION=1.1.0

RUN apk add --no-cache fontconfig curl ca-certificates \
    && curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C / \
    && npm install -g screenie-server@$SCREENIE_VERSION \
    && apk del curl \
    && rm -rf /tmp/*

EXPOSE 3000

CMD update-ca-certificates && /usr/local/bin/screenie
```

File: Dockerfile [hunk 1]
```diff
@@ -2,11 +2,14 @@ FROM node:7.7.2-alpine
 
 ENV SCREENIE_VERSION=1.1.0
 
+RUN apk --update upgrade && \
+    apk add curl ca-certificates
```
Comment (asteinlein) [inline Dockerfile]: What's curl needed for? The SO thread you linked to didn't mention that.

**Review by asteinlein — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you try the following and see if it still fixes your case:

Just add ca-certificates to the existing `apk add` line (instead of adding a separate RUN), and add a new line that runs `update-ca-certificates` into the existing RUN? Seems a bit overkill to run that command on every start of a container?

**Comment by asteinlein** [comment]
Basically, I like your minimal version better. Just wondering if we can run the `update-ca-certificates` command during build, instead of when starting the server.

File: Dockerfile (hunk 1 above)
Comment (anfedorov) [reply Dockerfile]: oops, that's from when I first tried adding it below. can take it out here.

**Review by anfedorov — COMMENTED** [review]
so certs can change at any time, and for some cases, you really want revoked certs to make it back to your images *before* they restart, and definitely before you re-build from your Dockerfile (which, if I understand correctly, is ideally never?).

happy to move it, but I'm pretty sure you'll run into the same case at some point unless regularly re-building the image from your Dockerfile is part of your process.

**Review by asteinlein — APPROVED** [review]

**Comment by asteinlein** [comment]
Thanks for the patch and the update! A new Docker image built from this is pushed to the Docker hub. :)

**Pull request merged**